### PR TITLE
NAACL 2019 proceedings

### PR DESCRIPTION
**Pull request by mjpost**
This will be merged first thing on June 2.

@davidweichiang, I have run `protect.py` on this, but what do you do for updating names?

### CODE REVIEW

**Comment by davidweichiang** [comment]
https://github.com/acl-org/acl-anthology/blob/auto_accents/bin/auto_authors.py

I don't think that's the very latest, though. I'll commit once the first run finishes successfully -- I'm doing everything from 2015-2019 and it goes very slowly because of downloading the PDFs. (And other problems, like power outages.)

**Comment by davidweichiang** [comment]
Also, protect.py should be deprecated in favor of normalize_anth.py, which does the same thing plus more.

**Comment by mjpost** [comment]
Great, thanks. I'll fix that later. The build (nicely) failed because of our favorite ambiguous author.

**Comment by mjpost** [comment]
Am I understanding correctly that you will apply `auto_authors.py` to the NAACL files?

Also, the first time I run `normalize_anth.py`, I should use `-t`?

**Comment by davidweichiang** [comment]
The current run of auto_authors.py does not include NAACL 2019, no (it's running on the master branch and started a few days ago). I can start a second run on NAACL 2019, but it needs the PDF files. Perhaps I should modify auto_authors.py to read PDF files locally, and you can run it, which would be much, much faster.

Yes, I think you should use the -t option to normalize_anth.py, since what comes out of START is basically TeX.

**Comment by mjpost** [comment]
Can someone approve this so I can merge it tomorrow morning (first thing June 2 AOE)?

File: data/xml/W19.xml [hunk 1]
```diff
@@ -1553,4 +1553,4215 @@
...
+    <title>Parallelizable Stack Long Short-Term Memory</title>
+    <author><first>Shuoyang</first><last>Ding</last></author>
+    <author><first>Philipp</first><last>Koehn</last></author>
+    <booktitle>Proceedings of the Third Workshop on Structured Prediction for <fixed-case>NLP</fixed-case></booktitle>
+    <month>June</month>
+    <year>2019</year>
+    <address>Minneapolis, Minnesota</address>
+    <publisher>Association for Computational Linguistics</publisher>
+    <pages>1–6</pages>
+    <abstract>Stack Long Short-Term Memory (StackLSTM) is useful for various applications such as parsing and string-to-tree neural machine translation, but it is also known to be notoriously difficult to parallelize for GPU training due to the fact that the computations are dependent on discrete operations. In this paper, we tackle this problem by utilizing state access patterns of StackLSTM to homogenize computations with regard to different discrete operations. Ou
```
Comment (mbollmann) [inline data/xml/W19.xml]: This line produces a warning during the build due to the usage of `\mathit`. That means it'll show the raw TeX command on the website. I should probably add some code to handle this in `texmath.py`, it's pretty trivial...

**Comment by mbollmann** [comment]
@mjpost, no bibliography files whatsoever have been generated for the website. Did you `make clear` before building? As long as the Makefile doesn't explicitly model dependencies, that's probably necessary.

**Comment by mjpost** [comment]
Oh, drat. Rebuilding now.

**Comment by mjpost** [comment]
Fixed.

**Pull request merged**